src/App.tsx

```diff
@@ -1,20 +1,23 @@
-import { PeoplePage } from './components/PeoplePage';
-import { Navbar } from './components/Navbar';
-
 import './App.scss';
+import { Navigate, Route, Routes } from 'react-router-dom';
+import Layout from './components/Layout';
+import HomePage from './pages/HomePage';
+import PageNotFound from './pages/PageNotFound';
+import PeoplePage from './pages/PeoplePage';
 
 export const App = () => {
   return (
     <div data-cy="app">
-      <Navbar />
-
-      <div className="section">
-        <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
-        </div>
-      </div>
+      <Routes>
+        <Route path="/" element={<Layout />}>
+          <Route index element={<HomePage />} />
+          <Route path="people" element={<PeoplePage />}>
+            <Route path=":slug" element={<PeoplePage />} />
```

Comment:
Using the same `PeoplePage` component for both the base path `/people` and the dynamic path `/:slug` might not be ideal. Consider using a separate component for the dynamic route to ensure proper handling of the dynamic content.